lib/Doctrine/Common/Annotations/CachedReader.php

```diff
@@ -251,13 +262,17 @@ private function getTraitLastModificationTime(ReflectionClass $reflectionTrait)
     {
         $fileName = $reflectionTrait->getFileName();
 
+        if (isset($this->loadedFilemtimes[$fileName])) {
+            return $this->loadedFilemtimes[$fileName];
+        }
+
         $lastModificationTime = max(array_merge(
             [$fileName ? filemtime($fileName) : 0],
             array_map([$this, 'getTraitLastModificationTime'], $reflectionTrait->getTraits())
         ));
 
         assert($lastModificationTime !== false);
 
-        return $lastModificationTime;
+        return $this->loadedFilemtimes[$fileName] = $lastModificationTime;
```

Comment:
mind the typo/cases: `$fileName` vs `$filename` and `$lastModification` vs `lastModificationTime`.

Not cool to homogenize it in a feature PR I guess.

Reply:
It is cool as long as you do it in a separate commit